rotkehlchen/chain/evm/decoding/curve_lend/utils.py

```diff
@@ -85,6 +85,15 @@ def _process_curve_lending_vault(database: 'DBHandler', vault: dict[str, Any]) -
         )],
         encounter=TokenEncounterInfo(description='Querying Curve lending vaults', should_notify=False),  # noqa: E501
     )
+    if (gauge_address := vault.get('gaugeAddress')) is not None:
+        get_or_create_evm_token(
+            userdb=database,
```

Comment:
do we need to invalidate the last time queried somehow?

Reply:
this wasn't answered

Reply:
yeah, added that to the global db migration or do you mean something else?